http/src/main/scala/plans/Blueprints.scala

```diff
@@ -147,7 +147,10 @@ final case class Blueprints(config: NelsonConfig) extends Default {
     case req @ POST -> Root / "v1" / "blueprints" & IsAuthenticated(session) if IsAuthorized(session) =>
       decode[BlueprintRequestJson](req){ bpr =>
         if (hasIntegrity(bpr.sha256, bpr.template))
-          json(Nelson.createBlueprint(bpr.name, bpr.description, bpr.sha256, bpr.template))
+          if (Option(bpr.name).exists(_ != ""))
```

Comment:
I would suggest doing something like this instead:

`Option(bpr.name).exists(_.trim.nonEmpty)`

This way you can account for more than the empty string (e.g. newlines, tabs, etc).

Reply:
This is a good suggestion, thanks!